### PR TITLE
EWL-4190 paragraph atom

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

Please do not submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.
**Jira Ticket**

- [EWL-4190 SG2 | Migrate paragraph atom](https://issues.ama-assn.org/browse/EWL-4190)

## Description

Adds paragraph atom

## To Test

- [ ] gulp serve

## Visual Regressions

Does it match SG1?

## Relevant Screenshots/GIFs
screen shot 2017-12-07 at 12 21 17 pm

## Remaining Tasks
n/a

## Additional Notes
n/a

### CODE REVIEW

File: styleguide/source/_patterns/01-atoms/paragraph.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "paragraph" : {
+    "text": "A paragraph (from the Greek paragraphos, \"to write beside\" or \"written beside\") is a self-contained unit of a discourse in writing dealing with a particular point or idea. A paragraph consists of one or more sentences. Though not required by the syntax of any language, paragraphs are usually an expected part of formal writing, used to organize longer prose."
+  }
+}
```
Comment (labbydev) [inline styleguide/source/_patterns/01-atoms/paragraph.json]: line ending

File: styleguide/source/_patterns/01-atoms/paragraph.twig [hunk 2]
```diff
@@ -0,0 +1 @@
+<p>{{paragraph.text | raw}}</p>
```
Comment (labbydev) [inline styleguide/source/_patterns/01-atoms/paragraph.twig]: Line ending

**Comment by designerneil** [comment]
@chelsiejohnston I don't know how you would be able to seperate header from headers or make it clearer. Header is for one header ie. h1, h2, h3. Headers shows all the variants we have on one page.

**Comment by designerneil** [comment]
@labbydev I've added reset.css and addressed the blank line comment

**Pull request merged**